CONTRIBUTING.md

```diff
@@ -0,0 +1,33 @@
+# Contributing
+
+## Before You Start Coding
+
+- Set up `git`, `node.js` and this project as described in [the README](./README.md)
+- Create a new branch with the name `<your-name>/<feature>` with `git checkout -b <your-name>/<feature>`
+- Style note: Branch names should be short. For instance something like `vivianwilde/contributing-guide` would be a good branch name
+- Make your changes to the code, browse stackoverflow in despair, etc. etc. (basically, the usual development steps)
+- Test your code, both by inspecting the running site and with automated tests (TODO: Our testing framework and guidelines)
+
+## Once Your Code is Ready
+
+- Once you're confident your code is working correctly:
```

Comment:
Add instructions for them to git status and git log to check if they need to pull or fetch before committing.

Reply:
I don't think this is required, since they'll be working on their own branch so it shouldn't have been modified by anyone other than them.